imap_processing/mag/l1a/mag_l1a.py

```diff
@@ -17,14 +20,115 @@ def mag_l1a(packet_filepath):
     packet_filepath :
         Packet files for processing
     """
-    mag_l0 = decom_mag.decom_packets(packet_filepath)
+    packets = decom_mag.decom_packets(packet_filepath)
 
-    mag_norm, mag_burst = decom_mag.export_to_xarray(mag_l0)
+    norm_data = packets["norm"]
+    burst_data = packets["burst"]
 
-    if mag_norm is not None:
-        file = write_cdf(mag_norm)
-        logger.info(f"Created CDF file at {file}")
+    if norm_data is not None:
+        mag_norm_raw = decom_mag.generate_dataset(
+            norm_data, mag_cdf_attrs.mag_l1a_norm_raw_attrs.output()
+        )
+        file = write_cdf(mag_norm_raw)
```

Comment:
If you are writing CDF file here, are you tracking output filepath somewhere that upload API can use later in `cli.py`?

Reply:
You're right that this needs to happen, I'll figure it out and put it in the CDF PR (incoming soon)

Reply:
sounds good!